protocols/floodsub/src/layer.rs

```diff
@@ -60,12 +60,21 @@ pub struct Floodsub<TSubstream> {
     marker: PhantomData<TSubstream>,
 }
 
+/// Configuration for the floodsub behaviour.
+#[derive(Debug, Clone)]
+pub struct FloodsubConfig {
+    /// Identifier of the local node.
+    pub local_peer_id: PeerId,
+    /// Maximum number of bytes allowed in an RPC message.
+    pub max_message_size: usize,
```

Comment:
Should these fields really be public? This could easily break clients in the future.